server2/Arcadia.Assistant/Arcadia.Assistant.ExternalStorages.SharepointOnline/SharepointApiModels/SharepointListItemsResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Arcadia.Assistant.ExternalStorages.SharepointOnline.SharepointApiModels
+{
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    public class SharepointListItemsResponse
+    {
+        [JsonPropertyName("value")]
```

Comment:
And here.

Reply:
No way to use Data serialization attribute here.